dbt_copilot_helper/utils/validation.py

```diff
@@ -17,6 +18,62 @@ def validator(string):
     return validator
 
 
+def validate_s3_bucket_name(name):
+    return False
+
+
+def validate_addons(addons: dict):
+    """
+    Validate the addons file and return a dictionary of addon: error message.
+    """
+    errors = {}
+
+    for addon_name, addon in addons.items():
+        try:
+            addon_type = addon.get("type", None)
+            if not addon_type:
+                errors[addon_name] = f"Missing addon type in addon '{addon_name}'"
+                continue
+            schema = SCHEMA_MAP.get(addon_type, None)
+            if not schema:
+                errors[
+                    addon_name
+                ] = f"Unsupported addon type '{addon_type}' in addon '{addon_name}'"
+                continue
+            schema.validate(addon)
+        except SchemaError as ex:
+            errors[addon_name] = f"Error in {addon_name}: {ex.code}"
+
+    return errors
+
+
+def int_between(lower, upper):
+    def is_between(value):
+        if isinstance(value, int) and lower <= value <= upper:
+            return True
+        raise SchemaError(f"should be an int between {lower} and {upper}")
```

Comment:
```suggestion
        raise SchemaError(f"Should be an integer between {lower} and {upper}")
```

Reply:
I'll keep the "s" of "should" lower case to be consistent with the built in errors.